src/main.rs

```diff
@@ -17,9 +20,11 @@ fn main() {
             ..Default::default()
         })
         .add_plugins(DefaultPlugins)
-        .add_system(bevy::input::system::exit_on_esc_system.system())
-        //.add_plugin(console::ConsolePlugin)
+        //.add_system(bevy::input::system::exit_on_esc_system.system())
```

Comment:
Let's keep these two systems in, they're pretty useful

Reply:
Actually i'm using escape to close the window, however i can link this system to the MainGame State